### DISCUSSION TITLE
If you re-organize your originals folder are there duplicate sidecar files, database entries, etc?

### DISCUSSION THREAD

**Discussion by miguelarios** [photoprism/photoprism#1435] [General]
I reorganized my photos in originals to basically move all photos I had in a YYYY/MM format to mobile/YYYY/MM format now, I am reindexing but wondering if I should purge everything and start again? Not really using much of tagging and such yet. Still figuring out stuff so not afraid to lose anything in terms of photoprism just concerned with duplicate files being created in my appdata folder and db.

**Comment by graciousgrey** [comment]
If you just moved your files without editing them, PhotoPrism recognizes their new location during the next index process and moves the related sidecar files so that there should not be duplicates.
